src/DRAMPower/DRAMPower/standards/lpddr5/LPDDR5.cpp

```diff
@@ -11,7 +11,7 @@ namespace DRAMPower {
 	LPDDR5::LPDDR5(const MemSpecLPDDR5 &memSpec)
 		: memSpec(memSpec)
 		, ranks(memSpec.numberOfRanks, { (std::size_t)memSpec.numberOfBanks })
-		, commandBus{6}
+		, commandBus{7}
 		, readBus{6}
```

Comment:
Why are readBus and writeBus initialized with 6?

Reply:
No idea, fixed it do 16